test/weex/cases/render/class.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <div :class="class1"></div>
+    <div :class="class2"></div>
+    <div class="box" :class="class3"></div>
+    <div class="box" :class="class4"></div>
```

Comment:
It's would be better if you add more inline statement examples in the test case, such as `:class="{ active: isActive }"`.